src/render/draw_custom.js

```diff
@@ -19,6 +19,7 @@ function drawCustom(painter: Painter, sourceCache: SourceCache, layer: CustomSty
         const prerender = implementation.prerender;
         if (prerender) {
             painter.setCustomLayerDefaults();
+            context.setColorMode(painter.colorModeForRenderPass());
```

Comment:
Nit: this isn't a docs change.

Reply:
Ahh, woops. I was going to commit that separately. This adds a default blending value for `prerender` so that it is more consistent with `render`. Should I split this into a separate commit?